workbench-core/authorization/src/caslAuthorizationPlugin.ts

```diff
@@ -38,4 +64,29 @@ export default class CASLAuthorizationPlugin implements AuthorizationPlugin {
     });
     return new Ability(rules);
   }
+  /**
+   * Given a set of {@link IdentityPermission}s, a CASL {@link Ability} is created.
+   * @param identityPermissions - a set of {@link IdentityPermission}s.
+   * @returns a CASL {@link Ability}
+   */
+  private _defineAbilitiesForIdentityPermissions(identityPermissions: IdentityPermission[]): Ability {
+    const { can, cannot, rules } = new AbilityBuilder(Ability);
+
+    identityPermissions.forEach((identityPermission) => {
+      const conditions = {
+        ...(identityPermission.conditions ?? {}),
+        ...(identityPermission.subjectId === '*' ? {} : { subjectId: identityPermission.subjectId })
+      };
+      if (identityPermission.effect === 'ALLOW')
+        can(identityPermission.action, identityPermission.subjectType, identityPermission.fields, conditions);
+      else
```

Comment:
Is it possible to do an early return here so we don't need have an `else` clause?

Reply:
sure, is there a benefit to eliminating the else clause?

Reply:
cyclomatic complexity management. `else` statements aren't really necessary if your code can be terminated inside an if clause.

Reply:
isn't the CYC measurement here the same with or without the else clause

Reply:
It is, but this is more about getting out of the practice of using else clauses that eventually would lead us down to high CYC. But like I was saying on the call, we can always refactor if it gets to that point in the future.